app/repositories/models.py

```diff
@@ -1,37 +1,47 @@
 from datetime import datetime
-
 from app.plugins import db
 
-
-class Order(db.Model):
-    _id = db.Column(db.Integer, primary_key=True)
-    client_name = db.Column(db.String(80))
-    client_dni = db.Column(db.String(10))
-    client_address = db.Column(db.String(128))
-    client_phone = db.Column(db.String(15))
-    date = db.Column(db.DateTime, default=datetime.utcnow)
-    total_price = db.Column(db.Float)
-    size_id = db.Column(db.Integer, db.ForeignKey('size._id'))
-
-    size = db.relationship('Size', backref=db.backref('size'))
-    detail = db.relationship('OrderDetail', backref=db.backref('order_detail'))
-
-
 class Ingredient(db.Model):
     _id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(80), nullable=False)
     price = db.Column(db.Float, nullable=False)
 
+class Beverage(db.Model):
+    _id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(80), nullable=False)
+    price = db.Column(db.Float, nullable=False)
 
 class Size(db.Model):
     _id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(80), nullable=False)
     price = db.Column(db.Float, nullable=False)
 
-
-class OrderDetail(db.Model):
+class OrderXIngredient(db.Model):
     _id = db.Column(db.Integer, primary_key=True)
-    ingredient_price = db.Column(db.Float)
     order_id = db.Column(db.Integer, db.ForeignKey('order._id'))
     ingredient_id = db.Column(db.Integer, db.ForeignKey('ingredient._id'))
+    order = db.relationship('Order', backref=db.backref('order_ingredients', overlaps="order,order_ingredients"))
     ingredient = db.relationship('Ingredient', backref=db.backref('ingredient'))
+
+class OrderXBeverage(db.Model):
+    _id = db.Column(db.Integer, primary_key=True)
+    order_id = db.Column(db.Integer, db.ForeignKey('order._id'))
+    beverage_id = db.Column(db.Integer, db.ForeignKey('beverage._id'))
+    order = db.relationship('Order', backref=db.backref('order_beverages', overlaps="order,order_beverages"))
+    beverage = db.relationship('Beverage', backref=db.backref('beverage'))
+
```

Comment:
What changes does the new schema imply?

Reply:
The new Schema implies two many-to-many relationships all connected to the order. Also, the order detail doesnt exist anymore, since it seemed unnecesary to have an order detail with only the ingredients outside, but the size inside. It would be either make it all part of it, or all outside. Making it all part of it doesn't make much sense either, since making a one-to-one table and doing changes based on it for the code can be a hassle and doesn't actually give a better performance. Also it makes sense for the ingredients, size and the beverage to be part of the order itself.